webpack.config.js

```diff
@@ -171,6 +171,7 @@ const config = {
     ].filter(Boolean),
     resolve: {
         alias: {
+            'focus-visible': `${__dirname}/node_modules/focus-visible/dist/focus-visible.min.js`,
```

Comment:
is the full path necessary? jquery seems to use a relative path just fine.

Reply:
That's correct! Updated